mako.systemd.service.in

```diff
@@ -6,7 +6,7 @@ PartOf=graphical-session.target
 [Service]
 Type=dbus
 BusName=org.freedesktop.Notifications
-ExecCondition=/bin/sh -c '[ "$XDG_SESSION_TYPE" == "wayland" ]'
+ExecCondition=/bin/sh -c '[ -z $WAYLAND_DISPLAY ] && exit 1 || exit 0'
```

Comment:
A cleaner way do this in systemd v246 and later is to use `ConditionEnvironment=WAYLAND_DISPLAY` instead.

Reply:
Nice! Care to make a PR with this change?

Reply:
Done. https://github.com/emersion/mako/pull/319